git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -0,0 +1,810 @@
+```
+
+When we run `git status` now,
+it tells us that a file it already knows about has been modified:
+
+```
+$ git status
+
+On branch main
+Changes not staged for commit:
+  (use "git add <file>..." to update what will be committed)
+  (use "git checkout -- <file>..." to discard changes in working directory)
+
+	modified:   mars.txt
+
+no changes added to commit (use "git add" and/or "git commit -a")
+```
+
+The last line is the key phrase:
+"no changes added to commit".
+We have changed this file,
+but we haven't told Git we will want to save those changes
+(which we do with `git add`)
+nor have we saved them (which we do with `git commit`).
+So let's do that now. It is good practice to always review
+our changes before saving them. We do this using `git diff`.
+This shows us the differences between the current state
+of the file and the most recently saved version:
+
+```
+$ git diff
+
+diff --git a/mars.txt b/mars.txt
+index df0654a..315bf3a 100644
+--- a/mars.txt
++++ b/mars.txt
+@@ -1 +1,2 @@
+ Cold and dry, but everything is my favorite color
++The two moons may be a problem for Wolfman
+```
+
+<div class = 'care'>
+The output is cryptic because it is meant for a computer, not a human.
+It is actually a series of commands for tools like editors and `patch`
+telling them how to reconstruct one file given the other. Nevertheless we can gain some useful information
+if we break it down into pieces:
+
+  1. The first line tells us that Git is comparing the old and new versions of the file `mars.txt`.
+  2. The second line tells exactly which versions of the file Git is comparing; `df0654a` and `315bf3a` are unique computer-generated labels for those versions.
+  3. The third and fourth lines once again show the name of the file being changed.
+  4. The remaining lines are the most interesting, they show us the actual differences and the lines on which they occur. In particular, the `+` marker in the first column shows where we added a line.
+</div>
+
+<div class = 'warning'>
+If we immediately commit this change using `git commit` what will happen?
+
+Git won't commit because we didn't use `git add` first! Git will give you a message reminding you to use `git add`.
+
+</div>
+
+
+Let's try adding the file and then committing the changes:
+
+```
+$ git add mars.txt
+$ git commit -m "Add concerns about effects of Mars' moons on Wolfman"
+
+[main 34961b1] Add concerns about effects of Mars' moons on Wolfman
+ 1 file changed, 1 insertion(+)
+```
+
+Double check that everything worked by running `git status`:
+
+```
+$ git status
+
+On branch main
+
+nothing to commit, working directory clean
+```
+
+### How often should you `commit`?
+
+Just like any method of backing up or saving your work, the benefits of using Git for version control are only available if you regularly use them. In our example, we have been committing every time we add or change a line in a single file. That is going to be odious overkill for most projects.
```

Comment:
```suggestion
Just like any method of backing up or saving your work, the benefits of using Git for version control are only available if you regularly use them. In our example, we have been committing every time we add or change a line in a single file. That is going to be odious overkill for most projects.  

Often, when we're working in a project, we change several files in the course of a few minutes or hours of work.  For example, you might change the counts of research subjects in a couple of different places across two or three files related to a research project when you update the weekly numbers.  It's helpful to know that with git, you can change a few files in your repository, then use `git add` to include the changed files (as many as you want) you want to include in a commit, and then do a single commit which explains all of the changes as a whole.
```

Trying to give an example and make it clear that commit can apply across multiple files...